assets/js-translations/exercise.fi.json

```diff
@@ -0,0 +1,4 @@
+{
+  "There will be an exercise when the module opens for submissions.":
+  "Tähän tulee tehtävä kierroksen avautuessa."
```

Comment:
The Finnish string is OK, but the English string requires improvement.

Reply:
New ideas:

Tässä julkaistaan tehtävä kierroksen avautuessa.

An exercise will be published here once the round opens for submissions.

Reply:
I approve these.